docs/third_party.md

```diff
@@ -66,6 +66,7 @@ These are projects we know about implementing Protocol Buffers for other program
 * Javascript: https://github.com/seishun/node-protoc-plugin (Node.js port of plugin.h)
 * Javascript: https://github.com/seishun/node-protoc-gen-javascript (Node.js port of the Google-official implementation)
 * Javascript: https://github.com/ButterCam/sisyphus-js
+* Typescript: https://github.com/thesayyn/protoc-gen-ts
```

Comment:
@thesayyn Would you mind moving this line below under Swift just to keep this alphabetically sorted?

Reply:
@acozzette Done 😇